modules/claims_api/app/mailers/claims_api/unsuccessful_report_mailer.rb

```diff
@@ -12,16 +12,15 @@ class UnsuccessfulReportMailer < ApplicationMailer
       premal.shah@va.gov
       lee.deboom@oddball.io
       dan.hinze@adhocteam.us
-      seth.johnson@gdit.com
-      kayur.shah@gdit.com
-      tim.barto@gdit.com
       zachary.goldfine@va.gov
     ].freeze
```

Comment:
i cant find it now, but i remember a conversation the other day where Emily Goodrich was asking what report we were all talking about and seemed interested in getting it herself. not sure if worth adding her email here or not.

Reply:
Thanks for letting me know, I'll add her.